### PR TITLE
ISSUE-28 add loading message

### LINKED ISSUE #28: Extension is unusable until the page loads
On slower pages such as <http://www.dailymail.co.uk/home/index.html> it is quite likely that a user will click on the extension before the page completes loading. In this case there is no Start button and no indication why. The start button does not appear when the page does load, giving the impression that the extension is broken.

### PR DESCRIPTION
**Pull request by tabanliviu**
fixes #28

### CODE REVIEW

**Comment by Ian-Hi** [comment]
Verified ok

File: app/popup.html [hunk 1]
```diff
@@ -44,6 +44,7 @@
       <div class="popup-colourscale-scale"></div>
     </div>
     <div class="popup-bottom">
+      <span id="loading">Please wait while the page loads...</span>
```
Comment (MalcolmSmithUK) [inline app/popup.html]: @tabanliviu INFO - Consider using `&hellip;` here...

File: app/popup.js [hunk 2]
```diff
@@ -32,16 +33,22 @@
 
   sendMessageToActiveTab({message: 'info'}, (response) => {
     if (response.message === 'inactive') {
+      getLoading().style.display = 'none';
       getStartButton().style.display = 'inline-block';
       getStopButton().style.display = 'none';
     }
 
     if (response.message === 'active') {
+      getLoading().style.display = 'none';
```
Comment (MalcolmSmithUK) [inline app/popup.js]: @tabanliviu MAINTAINABILITY - Please refactor to use 'hide' so you don't need any special knownledge of layout in your JS.

**Review by MalcolmSmithUK — CHANGES_REQUESTED** [review]

**Comment by tabanliviu** [comment]
@MalcolmSmithUK ready for review

**Review by MalcolmSmithUK — APPROVED** [review]

**Pull request merged**